concent_api/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Django >= 1.11, < 1.12
 psycopg2
+-e git://github.com/golemfactory/golem-messages.git@master#egg=golem_messages
```

Comment:
I see there are tagged releases now so let's switch from `master` to `v1.1.0`.

Don't forget to regenerate `concent_api/requirements.lock` after you change that.